sections/map_elements.tex

```diff
@@ -26,10 +26,19 @@ \subsection*{Map Tile Anatomy}\index{Map Tile}
 }
 \columnbreak
 
-\includegraphics[width=\linewidth]{\images/maptiles.png}
 \begin{center}
+  \begin{footnotesize}
```

Comment:
footnotesize is now done inside encircle already (in the bf branch...)

Reply:
Yes, this here is mostly for consistent look.